docs/sources/panels/visualizations/state-timeline.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "State timeline"
+description = "Stat timeline visualization docs"
+keywords = ["grafana", "docs", "state timeline", "panel"]
+weight = 900
++++
+
+# State timeline visualization
+
+This visualization excel at showing discrete state changes over time. Each field or series is rendered as it's own unique horizontal band. State regions can either be rendered with values or without. This panel works great with string or boolean states but can also be used with time series. When used with time series the thresholds are used to turn the numerical values into discrete state regions.
+
+{{< figure src="/static/img/docs/v8/state_timeline_strings.png" max-width="1025px" caption="state timeline with string states" >}}
+
+## Shared options
+
+- [Standard options]({{< relref "../standard-options.md" >}}) (Unit, min, max, decimals, color)
+- [Thresholds]({{< relref "../thresholds.md" >}})
+- [Value mappings]({{< relref "../value-mappings.md" >}})
+
+You can also use [field overrides]({{< relref "../field-overrides.md" >}}) to specify options per field or series.
+
+## Display options
+
+| Name           | Description                                                                                             |
+| -------------- | ------------------------------------------------------------------------------------------------------- |
+| `Show values`  | Controls whether values are rendered inside the state regions. Auto will render values if there is room |
+| `Align values` | Controls value alignment inside state regions.                                                            |
+| `Row height`   | Controls how much space between rows there are. 1 = no space = 0.5 = 50% space                          |
+| `Line width`   | Controls line width of state regions                                                         |
+| `Fill opacity` | Controls the opacity of state regions                                                         |
+
+## Value mappings
+
+To assign colors to boolean or string values use the [Value mappings](< {{ refref "../value-mappings.md"}} >).
+
+{{< figure src="/static/img/docs/v8/value_mappings_side_editor.png" max-width="300px" caption="Value mappings side editor" >}}
+
+## Time series + thresholds
+
+The panel can be used to great effect with time series as well. In this scenario the thresholds will be used to turn the time series into discrete colored state regions.
+
+{{< figure src="/static/img/docs/v8/state_timeline_time_series.png" max-width="1025px" caption="state timeline with time series" >}}
+
+## Legend options
+
+When the legend is enabled it can can show either the value mappings or the threshold brackets. To show the value mappings in the legend it's important that the `Color scheme` option under [Standard options]({{< relref "../standard-options.md" >}}) is set to `Single color` or `Classic palette`. To see the threshold brackets in the legend set the `Color scheme` to `From thresholds`.
```

Comment:
```suggestion
When the legend option is enabled it can show either the value mappings or the threshold brackets. To show the value mappings in the legend, it's important that the `Color scheme` option under [Standard options]({{< relref "../standard-options.md" >}}) is set to `Single color` or `Classic palette`. To see the threshold brackets in the legend set the `Color scheme` to `From thresholds`.
```